tests/data/state_estimation/inf-measurement-with-injection/params.json

```
@@ -0,0 +1,18 @@
+{
+  "calculation_method": [
+    "iterative_linear",
+    "newton_raphson"
+  ],
+  "rtol": 1e-8,
+  "atol": {
+    "default": 1e-5
+  },
```

shouldn't this just be like before? `extra_params` is a patch over the original data, so you don't need to explicitly specify a `default`, right?

```suggestion
  "atol": 1e-5
```